package.json

```diff
@@ -38,13 +45,20 @@
     "eslint-plugin-playwright": "^0.15.3",
     "eslint-plugin-react": "7.32.2",
     "eslint-plugin-react-hooks": "4.6.0",
-    "jest": "^29.4.1",
-    "jest-environment-jsdom": "^29.4.1",
+    "happy-dom": "~9.20.3",
     "nx": "19.0.3",
+    "nx-stylelint": "^17.1.4",
     "prettier": "^2.6.2",
     "sass": "1.62.1",
-    "ts-jest": "^29.1.0",
+    "stylelint": "^15.0.0",
+    "stylelint-config-standard": "^34.0.0",
     "ts-node": "10.9.1",
-    "typescript": "~5.4.2"
+    "typescript": "~5.4.2",
+    "vite": "~5.0.0",
+    "vitest": "^1.3.1"
+  },
+  "engines": {
+    "node": "20",
```

Comment:
what is the current Node version for the app?
My recommendation would be v20.13.1 as its currently LTS

Reply:
yeah I'll ensure on a specific version thanks